postman/collections/graviteeio-am-scim-collection-app-version.json

```diff
@@ -2014,519 +1961,1809 @@
 									},
 									"response": []
 								}
-							],
-							"protocolProfileBehavior": {},
-							"_postman_isSubFolder": true
+							]
 						},
 						{
-							"name": "List",
+							"name": "Patch",
 							"item": [
 								{
-									"name": "List users",
+									"name": "Invalid request - user not found",
 									"event": [
 										{
 											"listen": "test",
 											"script": {
-												"id": "bb757cde-314b-438d-a69e-75bbb80ec32e",
 												"exec": [
-													"pm.test(\"Status code is 200\", function () {",
-													"    pm.response.to.have.status(200);",
+													"pm.test(\"Status code is 404\", function () {",
+													"    pm.response.to.have.status(404);",
 													"});",
 													"",
-													"// 2 users (one from SCIM and an other from inline idp)",
-													"pm.test(\"SCIM list users\", function () {",
+													"pm.test(\"SCIM patch user unknown user\", function () {",
 													"    var jsonData = pm.response.json();",
-													"    pm.expect(jsonData.schemas).to.eql(['urn:ietf:params:scim:api:messages:2.0:ListResponse']);",
-													"    pm.expect(jsonData.totalResults).to.eql(1);",
-													"    pm.expect(jsonData.startIndex).to.eql(1);",
-													"    pm.expect(jsonData.itemsPerPage).to.eql(1);",
-													"    pm.expect(jsonData.Resources.length).to.eql(1);",
+													"    pm.expect(jsonData.status).to.eql(\"404\");",
+													"    pm.expect(jsonData.detail).to.eql(\"User [wrong-id] can not be found.\");",
+													"    pm.expect(jsonData.schemas).to.eql(['urn:ietf:params:scim:api:messages:2.0:Error']);",
 													"});"
 												],
 												"type": "text/javascript"
 											}
 										}
 									],
 									"request": {
-										"method": "GET",
+										"method": "PATCH",
 										"header": [
 											{
 												"key": "Authorization",
-												"value": "Bearer {{access_token}}",
+												"type": "text",
+												"value": "Bearer {{access_token}}"
+											},
+											{
+												"key": "Content-Type",
+												"name": "Content-Type",
+												"value": "application/json",
 												"type": "text"
 											}
 										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\n     \"schemas\":[\"urn:ietf:params:scim:api:messages:2.0:PatchOp\"],\n     \"Operations\": [{\n        \"op\":\"Replace\",\n        \"path\":\"active\",\n        \"value\": true\n     }]\n\n}"
+										},
 										"url": {
-											"raw": "{{gateway_url}}/{{domain}}/scim/Users",
+											"raw": "{{gateway_url}}/{{domain}}/scim/Users/wrong-id",
```

Comment:
Beware domainHrid

Reply:
Will be part of the Postman refactor PR.